### ISSUE TITLE
Private Rooms

### ISSUE THREAD

**Issue by r00tww** [quackduck/devzat#245]
Does Devzat supports secret rooms? Does it has password protected rooms?

**Comment by Arkaeriit** [comment]
Nope, you can host your own instances which may be whitelist-protected. But inside an instance, everyone can go to any room.

Still, if you want to do secret 1 on 1 conversation, you can used DMs.

**Comment by r00tww** [comment]
@Arkaeriit ok tysm

**Comment by quackduck** [comment]
Are password protected rooms a feature that you'd like?

**Comment by r00tww** [comment]
@quackduck yes.
